content/en/platform/corda/4.11/enterprise/network/available-network-parameters.md

```diff
@@ -11,46 +11,100 @@ title: Available network parameters
 weight: 37
 ---
 
-This topic lists the currently-available network parameters:
+This topic lists the currently-available network parameters, in alphabetical order:
 
-* **minimumPlatformVersion**:
-The minimum platform version that the nodes must be running. Any node which is below this will
-not start.
+## `confidentialIdentityMinimumBackupInterval`
+
+The `confidentialIdentityMinimumBackupInterval` network parameter is an optional parameter of type Duration. It specifies the minimum age of a generated Confidential Identity key before it can be used. This can be overridden in the node configuration or if a more recent database backup is indicated via RPC/shell. 

```

Comment:
I probably wouldn't specify the variable type as obviously as this, or do it for all parameters otherwise it sounds inconsistent, also its not quite clear how they can specify that duration (e.g. 1m or 1d etc)

Reply:
In this PR I only added a few parameters (the ledger recovery related ones), but restructured the doc so it looks like i worked on the other parameters. So I'm not sure what type those variables are. So I know it's inconsistent, but I'd prefer the new ones at least to be more accurate. What I can do is go through those parameters in a separate PR.

"its not quite clear how they can specify that duration"

I gave an example below this section using minutes - should I add another example for days?

Reply:
Ok makes sense, If there is an example, then that is fine. If you specify parameter types (I don't know if you have a standard way of doing that), I would make them code style so ...of type `Duration`. That is my personal preference for what I think looks better.